cpp/dolfinx/fem/Form.h

```diff
@@ -606,9 +606,18 @@ class Form
   // True if permutation data needs to be passed into these integrals
   bool _needs_facet_permutations;
 
-  // Entity maps (see Form documentation)
-  std::map<std::shared_ptr<const mesh::Mesh<geometry_type>>,
-           std::vector<std::int32_t>>
-      _entity_maps;
+  // Domain data for argument functions
+  //  [rank_i][IntegralType, integral(id), cell_type] -> entity map
```

Comment:
I think "-> entity map" might be a bit misleading here. This gives the mapped entities for the requested [rank_i][IntegralType, integral(id), cell_type]